demos/gfql/temporal_predicates.ipynb

```diff
@@ -0,0 +1,454 @@
+   "metadata": {},
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "source": "## 8. Using Wire Protocol Dictionaries\n\nYou can pass wire protocol dictionaries directly to temporal predicates. This is useful for:\n- Loading predicate configurations from JSON files\n- Building predicates programmatically\n- Sharing predicate definitions between systems",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "metadata": {},
+   "outputs": [],
+   "source": "# Complex multi-hop with temporal constraints\n# Find 2-hop paths through recent transactions\ntwo_hop_recent = g.chain([\n    # Start from business accounts\n    n(filter_dict={\"account_type\": \"business\"}),\n    # First hop: recent outgoing transactions\n    e_forward({\n        \"timestamp\": gt(datetime.now() - timedelta(days=30))\n    }, name=\"hop1\"),\n    # Intermediate nodes\n    n(),\n    # Second hop: any transaction\n    e_forward(name=\"hop2\"),\n    # Final nodes\n    n()\n])\n\nprint(f\"2-hop paths from business accounts through recent transactions:\")\nprint(f\"  Total edges: {len(two_hop_recent._edges)}\")\nprint(f\"  Hop 1 edges: {two_hop_recent._edges['hop1'].sum()}\")\nprint(f\"  Hop 2 edges: {two_hop_recent._edges['hop2'].sum()}\")",
+   "execution_count": 16
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "\n",
+    "This notebook demonstrated:\n",
+    "\n",
+    "1. **DateTime filtering** with `gt`, `lt`, `between` predicates on edges\n",
+    "2. **Date-only filtering** for day-level granularity on nodes\n",
+    "3. **Time-of-day filtering** for patterns like business hours\n",
+    "4. **Complex queries** combining temporal and non-temporal predicates\n",
+    "5. **Multi-hop queries** with temporal constraints using chain operations\n",
+    "6. **Temporal value classes** for explicit control\n",
+    "7. **Timezone-aware** filtering\n",
+    "8. **Proper chain syntax** with edge filters in `e_forward()` and node filters in `n()`\n",
+    "\n",
+    "Temporal predicates in GFQL provide a powerful way to analyze time-series aspects of graph data, enabling complex temporal queries while maintaining the expressiveness of graph traversals."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.0"
+  }
+ },
+ "nbformat": 4,
```

Comment:
Look at other IPYNB's, normally there's a CTA / what's next . Unclear what that would be for this temporal guide

* One  interesting one is our blogpost on visualizing time: https://www.graphistry.com/blog/unraveling-many-events-with-time-animated-graphs
* One of our radial layouts has a continuous time mode, so there's an ipynb there
* API calls for these too?
* GFQL getting started?